Common/src/Auth/Authorization/RequiresPermissionAttribute.cs

```diff
@@ -31,11 +31,17 @@ public class RequiresPermissionAttribute : AuthorizeAttribute
   {
     public const string PolicyPrefix = "RequiresPermission:";
 
-    public RequiresPermissionAttribute(string category, string function) => Permission = new Permissions.Permission(category, function);
+    public RequiresPermissionAttribute(string category,
+                                       string function)
+      => Permission = new Permissions.Permission(category,
+                                                 function);
 
-    public Permissions.Permission Permission
+    public Permissions.Permission? Permission
     {
-      get => Policy != null ? Permissions.Parse(Policy[PolicyPrefix.Length..]) : Permissions.None;
+      get
+        => Policy != null
+             ? Permissions.Parse(Policy[PolicyPrefix.Length..])
```

Comment:
I am not sure a lazy parsing is good. I think it might be better to parse once at the start.